rtdata/languages/default

```diff
@@ -1943,6 +1943,16 @@ PREFERENCES_LENSFUNDBDIR;Lensfun database directory
 PREFERENCES_LENSFUNDBDIR_TOOLTIP;Directory containing the Lensfun database.  Leave empty to use the default directories.
 PREFERENCES_LENSPROFILESDIR;Lens profiles directory
 PREFERENCES_LENSPROFILESDIR_TOOLTIP;Directory containing Adobe Lens Correction Profiles (LCPs)
+PREFERENCES_MAX_ZOOM_TITLE;Maximal zoom limit
```

Comment:
```suggestion
PREFERENCES_MAX_ZOOM_TITLE;Maximum zoom
```
"Maximum" is more commonly used in general than "Maximal" and describes the setting better. "limit" is redundant since "Maximum" is a limit.